quantmsio/core/psm.py

```
@@ -1,329 +1,201 @@
+                    row["scan_number"],
+                ],
+                axis=1,
             )
-            logger.info("The parquet file was generated in: {}".format(self.parquet_path))
-
+            map_dict = pep_msg.to_dict()["pep_msg"]
+            for key, value in map_dict.items():
+                if key not in pep_map:
+                    pep_map[key] = value
+                elif value[0] < pep_map[key][0]:
+                        pep_map[key] = value
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)